.github/renovate.json5

```diff
@@ -0,0 +1,66 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": [
+        "config:recommended",
+        ":dependencyDashboardApproval",
+        ":semanticCommitTypeAll(ci)"
+    ],
```

Comment:
@HonkingGoose I don't recall ever using `extends` before. The docs at https://docs.renovatebot.com/presets-config/#configrecommended don't say what the explicit values are. Is that intentional? If not, would a good next step be for me to file a docs issue?